packages/tools/devtools/devtools-example/jest.config.js

```diff
@@ -2,16 +2,28 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+// Get the test port from the global map and set it in env for this test
+const testTools = require("@fluidframework/test-tools");
+const { name } = require("./package.json");
+
+const mappedPort = testTools.getTestPort(name);
+process.env["PORT"] = mappedPort;
 
 module.exports = {
-	preset: "ts-jest",
+	preset: "jest-puppeteer",
 	roots: ["<rootDir>/src"],
+	globals: {
+		PATH: `http://localhost:${mappedPort}`,
+	},
 	transform: {
-		"^.+\\.tsx?$": "ts-jest",
+		"^.+\\.(ts|tsx)?$": "ts-jest",
 	},
-	testRegex: "(/__tests__/.*|(\\.|/)(test|spec))\\.(ts|tsx)?$",
+	testTimeout: 30_000,
+	testRegex: "(/__tests__/.*|(\\.|/)(test|spec))\\.ts?$",
 	testPathIgnorePatterns: ["/node_modules/", "dist"],
 	moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
 	coveragePathIgnorePatterns: ["/node_modules/", "/src/test/"],
-	testEnvironment: "jsdom",
+	// While we still have transitive dependencies on 'uuid<9.0.0', force the CJS entry point:
+	// See: https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library
+	moduleNameMapper: { "^uuid$": "uuid" },
```

Comment:
Is this needed? This package should be using common js already.

Reply:
Sorry, I see the change to ESNext here as well. I left a question about that, but I'll close this one.